Configurations/unix-Makefile.tmpl

```diff
@@ -686,8 +686,10 @@ tar:
 	DISTDIR=$(NAME); \
 	mkdir -p $$TMPDIR/$$DISTDIR; \
 	(cd $(SRCDIR); \
+	 submodule_re=`git submodule status | sed -e 's/^.//' | cut -d' ' -f2`; \
+	 submodule_re="^(`echo $$submodule_re | sed -e 's/ /|/g'`)\$$"; \
 	 git ls-tree -r --name-only --full-tree HEAD \
-         | grep -v '^fuzz/corpora' \
+	 | grep -v '^fuzz/corpora' | egrep -v "$$submodule_re" \
```

Comment:
> The submodule regexp is one ending in a `$`

But it doesn't have to, `^` leaves no ambiguity. As for -E, Tru64, HP-UX, AIX, Solaris 10, MacOS X, Linux from before 2000...